### PR TITLE
US111909 Properly update d2l-activities

### PR DESCRIPTION
**Pull request by ryantmer**
Somehow I managed to include an old version in #1820; this includes the most up-to-date `d2l-activities`, along with some other bumps.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3841,7 +3841,7 @@
       }
     },
     "d2l-enrollments": {
-      "version": "github:BrightspaceHypermediaComponents/enrollments#97722032486883be70cb4930350b045a344a933b",
+      "version": "github:BrightspaceHypermediaComponents/enrollments#49f18341cecbdd1ff421939c63923b40e2353003",
```
Comment (ryantmer) [inline package-lock.json]: FYI @AllanKerr

File: package-lock.json [hunk 2]
```diff
@@ -4191,7 +4191,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#ddcd30086d0c858756514de558b732f763024e52",
+      "version": "github:BrightspaceHypermediaComponents/organizations#de266a093b043008643d5b71aa2b74de2eadf2e4",
```
Comment (ryantmer) [inline package-lock.json]: FYI @m6jones @iFun

File: package-lock.json [hunk 3]
```diff
@@ -4255,7 +4255,7 @@
       }
     },
     "d2l-pdf-viewer": {
-      "version": "github:Brightspace/d2l-pdf-viewer#ced5da4147e1e119f2c3cc6d59690e2e5e626701",
+      "version": "github:Brightspace/d2l-pdf-viewer#3dd9feea4b2c525ff7726d29c79130a089d7398f",
```
Comment (ryantmer) [inline package-lock.json]: FYI @apalaniuk

File: package-lock.json [hunk 4]
```diff
@@ -3608,7 +3608,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#239300ed794782fe1a5aa8b210355fa3bce23320",
+      "version": "github:BrightspaceHypermediaComponents/activities#08290f5a6b56dba4d5d4c81f33d8c4588d442e89",
```
Comment (ryantmer) [inline package-lock.json]: FYI @cbusmand2l @m6jones @BhamelinD2L @dmackenz

File: package-lock.json (hunk 3 above)
Comment (apalaniuk) [reply package-lock.json]: Ack. The (non!) functional change there was 1. to make a large thing an optional dependency that shouldn't be installed here anyway, and 2. to magic-comment a Webpack thing (dynamic import parsing), which shouldn't have an effect here. But, I'll take a look!

**Pull request merged**